mlflow/store/file_store.py

```diff
@@ -140,15 +140,18 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
             rsl += self._get_deleted_experiments(full_path=False)
         return [self._get_experiment(exp_id, view_type) for exp_id in rsl]
 
-    def _create_experiment_with_id(self, name, experiment_id, artifact_uri):
+    def _create_experiment_with_id(self, name, experiment_id, artifact_uri, artifact_relative):
```

Comment:
Can you not achieve the same by setting store.artifact_root_uri to store. root_directory?